atlp-rwanda/atlp-pulse-fn
# Feature/notifications preference

# PR Description

Implementation of notification preference

# Description of tasks that were expected to be completed

We should be able to click and change push and email either on or off

# How has this been tested locally?

- Clone [repo](https://github.com/atlp-rwanda/atlp-pulse-fn)
- Change into the directory **atlp-pulse-fn**
- Checkout to the branch: [feature/notifications-preference](https://github.com/atlp-rwanda/atlp-pulse-fn/tree/feature/notifications-preference)
- Run the command `npm install` 
- Run the command `npm run dev`
- Visit ([Visit Preview](https://vercel.live/open-feedback/metron-devpulse-git-feature-notifications-preference-metron.vercel.app?via=pr-comment-visit-preview-link&passThrough=1))
- Login as trainee
- Enable push or email notification in settings
- Login as Coordinator and try give rating for push notification and updating rating for email notificatiom
- Then disable push or email notification and trigger notifications again to see if it's not sent

# Screenshot

<img width="1440" alt="Screenshot 2023-07-26 at 17 26 19" src="https://github.com/atlp-rwanda/atlp-pulse-fn/assets/107047654/3d0d849d-a662-4ffd-b2e5-ac748a3f17e9">
